docroot/modules/custom/foia_webform/src/Plugin/WebformHandler/FoiaSubmissionQueueHandler.php

```diff
@@ -32,7 +33,11 @@ public function postSave(WebformSubmissionInterface $webformSubmission, $update
       $componentAssociatedToWebform = $this->getComponentAssociatedToWebform($webformSubmission);
       if ($componentAssociatedToWebform) {
         $foiaRequest = $this->createFoiaRequest($webformSubmission, $componentAssociatedToWebform);
-        $this->queueFoiaRequest($foiaRequest);
+      }
+
+      if ($foiaRequest && $foiaRequest->getRequestStatus() === FoiaRequestInterface::STATUS_QUEUED) {
+        $queuer = new FoiaSubmissionQueueingService();
+        $queuer->addRequestToQueue($foiaRequest);
```

Comment:
I'm thinking these three lines of code can actually exist within the `queueFoiaRequest` method, so it would be:
```
if ($foiaRequest->getRequestStatus() === FoiaRequestInterface::STATUS_QUEUED) {
  $queuer = new FoiaSubmissionQueueingService();
  $queuer->addRequestToQueue($foiaRequest);
}
else {
  return FALSE;
}
```

or something like that. That way the code within the `postSave` method still reads as "create a request, queue a request"